### ISSUE TITLE
Hard coded path for dll's in project file

### ISSUE THREAD

**Issue by StompyNZ** [dmustanger/7dtd-ServerTools#12]
https://github.com/dmustanger/7dtd-ServerTools/blob/v3.7/ServerTools/ServerTools.csproj

This file has the reference dll's coded for a specific location

should be:
```
  <ItemGroup>
    <Reference Include="Assembly-CSharp">
      <HintPath>..\7dtd-binaries\Assembly-CSharp.dll</HintPath>
      <Private>False</Private>
    </Reference>
    <Reference Include="Assembly-CSharp-firstpass">
      <HintPath>..\7dtd-binaries\Assembly-CSharp-firstpass.dll</HintPath>
      <Private>False</Private>
    </Reference>
    <Reference Include="LogLibrary">
      <HintPath>..\7dtd-binaries\LogLibrary.dll</HintPath>
      <Private>False</Private>
    </Reference>
    <Reference Include="System" />
    <Reference Include="System.Xml" />
    <Reference Include="UnityEngine">
      <HintPath>..\7dtd-binaries\UnityEngine.dll</HintPath>
      <Private>False</Private>
    </Reference>
  </ItemGroup>
```
that way the files can be placed in the 7dtd-binaries folder as per the readme in that folder.

**Comment by dmustanger** [comment]
Will be fixed in 3.8

**Comment by WonderBoy-Ari** [comment]
Hey. Its Aribo. I know Jack About GIT. But Just wanted to show my support here too.
